### PR TITLE
Elasticsearch 7.2

### PR DESCRIPTION
**Pull request by Crell**
Pending confirmation from Engineering that it's fully tested.

### CODE REVIEW

File: src/changelog.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 ## 2019
 
 * **June 2019**
+  * Elasticsearch 7.2: We now support [Elasticsearch 7.2](/configuration/services/elasticsearc.md).
```
Comment (chadwcarlson) [inline src/changelog.md]: bad link - elasticsearch.md

File: src/changelog.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
 ## 2019
 
 * **June 2019**
+  * Elasticsearch 7.2: We now support [Elasticsearch 7.2](/configuration/services/elasticsearch.md).
```
Comment (chadwcarlson) [inline src/changelog.md]: If 5.2 and 5.4 are now deprecated, there should probably be an entry for them in the changelog too, like "[Elasticsearch](/configuration/services/elasticsearch.md) 5.2 and 5.4 support is now deprecated.

Comment (chadwcarlson) [reply src/changelog.md]: Plus I think June and July got switched, and this should be these should be the most recent additions in **July 2019**.

Comment (otaviojava) [reply src/changelog.md]: Fixed!

**Pull request merged**